superset-frontend/src/views/CRUD/data/database/DatabaseModal/styles.ts

```diff
@@ -156,7 +156,7 @@ export const antDModalStyles = (theme: SupersetTheme) => css`
   }
 
   .ant-modal-body {
-    height: ${theme.gridUnit * 180.5}px;
+    height: ${theme.gridUnit * 188}px;
```

Comment:
A good practice is to define numbers as a const with a comment providing its purpose, then use the const.  Referred to commonly as a "magic number" because there is no context to what the value means.  Having a const with a comment will help future developers understand what this number represents, and how changing it will affect the rendering of the component. 

I this case you would define a const outside of the style rendering block like:

`const DESCRIPTIVE_NAME:number = 188 // Description of what this is for...`

Then in the styling block use it like:
`height: ${theme.gridUnit * DESCRIPTIVE_NAME}px;`

I know you did not define this originally, but this is a simple opportunity to make code more readable for future devs :)

Reply:
I agree, and I changed it. Thank you!